Assets/Scripts/DayNightCycle/TimeStamps.cs

```diff
@@ -0,0 +1,93 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public static class TimeStamps
+{
+    public delegate void UpdateTime(int value);
+    public static UpdateTime hourTime;
+    public static UpdateTime minuteTime;
+    public static UpdateTime secondTime;
+    public static UpdateTime dayTime;
+    public static float wholeDayTime;
+    private static bool timeRunning;
```

Comment:
this should be settable, in case we want to stop time and currently in scene time isn't passing

Reply:
Indeed, my bad